README.md

```diff
@@ -34,21 +34,56 @@ Refer to these files by adding a `<link>` and a `<script>` element into your HTM
 
 ### Install with NPM
 
-If you have `node` installed on your machine, you can use `npm` to install the Web Design Standards. In your `package.json` list the following under `dependencies`:
+If you have `node` installed on your machine, you can use `npm` to install the
+Draft Web Design Standards.
 
-```json
-  "uswds": "git@github.com:18F/web-design-standards.git#v0.9.0"
 ```
+cd path/to/project-using-npm
+npm install --save uswds
+```
+
+This will add the Draft Web Design Standards as a dependency for your project and save
+it to your `package.json` file. The package will be installed in `node_modules`
+under `uswds`.
+
+Refer to these files by adding a `<link>` and a `<script>` element into your HTML pages:
 
-On subsequent runs of `npm install`, the package will be installed in `node_modules` under `uswds`. The CSS will be *not* compiled after it is installed. Instead, you are able to compile it however it fits into your project.
+```
+<link rel="stylesheet" href="./node_modules/uswds/assets/css/uswds.css">
+<link rel="stylesheet" href="./node_modules/uswds/assets/css/google-fonts.css">
+<!-- Include jQuery from the `uswds` project before `components.js` -->
+<script src="./node_modules/uswds/assets/js/vendor/jquery-1.11.3.min.js"></script>
```

Comment:
Yeah, I agree on the future thinking. I'm not quite sure the best way to recommend this. Above this line, I add the requirements about `jQuery` and `$` needing to be on the window object.